src/app/helpers/jwt.interceptor.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HTTP_INTERCEPTORS, HttpInterceptor, HttpHandler, HttpRequest ,HttpEvent } from '@angular/common/http';
+
+import { TokenStorageService } from '../services/token-storage.service';
+import { Observable } from 'rxjs';
+const TOKEN_HEADER_KEY = 'Authorization';  
+
+@Injectable()
+export class JWTInterceptor implements HttpInterceptor {
+  constructor(private tokenStorageService: TokenStorageService) { }
+
+  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    const token = this.tokenStorageService.getToken();
+
+    if (token != null) {
+        req = req.clone({
+            setHeaders: {
+               Authorization: "Bearer" + token,
```

Comment:
We need a space between Bearer and token
It would be better to write: \`Bearer ${token}\`